src/components/shared/TractionInput.vue

```diff
@@ -3,14 +3,9 @@
     <label v-if="title">{{ title }}</label>
     <input
       :id="id"
-      :value="content"
+      v-bind="$attrs"
```

Comment:
Does this work by spreading/binding all passed in props to this component? If so does that mean we still need to explicitly declare :value ?

Reply:
From the documentation I read, it says we need explicit binding for value field as this is a custom component wrapper.

Reply:
👍 Do you have a link to the doc?

Reply:
https://v2.vuejs.org/v2/guide/components.html#Using-v-model-on-Components
https://vuejs.org/guide/components/attrs.html
Also, if you search for Creating an Input component using Vue, you can find similar documentation - like https://dev.to/abhisheksarmah/creating-an-input-component-using-vue-and-tailwindcss-3a6b